mscore/musescore.cpp

```diff
@@ -246,6 +252,9 @@ const std::list<const char*> MuseScore::_basicNoteInputMenuEntries {
             "",
             "pad-rest",
             "",
+            "pad-raise",
+            "pad-lower",
+            "",
```

Comment:
*MuseScore is overloaded compared to Finale*
Let's not make any decision based on this opinion... as you say, it's not related to this PR and a lot of people would totally disagree.

According to me, MuseScore existed for 12 years without this 1/2 buttons, so it can exist 10 more years without them. Also, I can't remember of any user requesting this feature, finale switchers or not. I would just prefer to have them in the advanced workspace if we end up merging this request.